### PR TITLE
Clarify that "Stash Host" is a REST API URL, not a plain hostname

### PR DESCRIPTION
**Pull request by jimklimov**
Just a cosmetic change in README and GUI resources, but very helpful for first-time users of the plugin :)

### CODE REVIEW

**Comment by jimklimov** [comment]
Pinging @nemccarthy as the only maintainer, to get this PR moving forward :)

File: README.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ Select *Git* then configure:
 Select *Stash Pull Request Builder* then configure:
 
 - **Cron**: must be specified. eg: every 2 minute `H/2 * * * *`
-- **Stash Host**: the *http* or *https* URL of the Stash host (NOT *ssh*). eg: *https://example.com*
-- **Stash Credentials**: Select or Add the login username/password for the Stash Host
+- **Stash REST API Host (HTTP URL)**: the *http* or *https* URL of the Stash host (NOT *ssh*). eg: *https://example.com*
```
Comment (jakub-bochenski) [inline README.md]: Why not just `Stash URL: the *http* or *https* URL of the Stash instance (NOT *ssh*). eg: *https://example.com*`

**Comment by jakub-bochenski** [comment]
@jimklimov  Can we please drop the formatting changes and just limit this to the field description/documentation?

**Comment by jimklimov** [comment]
Separated the whitespace changes into #30

File: README.md (hunk 1 above)
Comment (jimklimov) [reply README.md]: Not sure in hindsight, but probably made sense to differentiate the REST API aspect from just the git protocol server that can be accessed over http(s) or ssh, and certainly to specify it is an URL and not just a hostname (which your suggestion fulfills too) :)

**Comment by jimklimov** [comment]
Updated the PR payload, and rebased over current master.

**Comment by proski** [comment]
Both commits touch the same lines in the same files. It would be better to merge them into one commit. Anyway, the change looks good to me.

**Review by jakub-bochenski — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -44,8 +44,8 @@ Select *Git* then configure:
 Select *Stash Pull Request Builder* then configure:
 
 - **Cron**: must be specified. eg: every 2 minute `H/2 * * * *`
-- **Stash Host**: the *http* or *https* URL of the Stash host (NOT *ssh*). eg: *https://example.com*
-- **Stash Credentials**: Select or Add the login username/password for the Stash Host
+- **Stash URL**: the *http* or *https* URL of the Stash REST API Host (NOT *ssh*). eg: *https://example.com*
```
Comment (jakub-bochenski) [inline README.md:47]: One more thing we could clarify would be what if your stash is not at host root, e.g. https://example.com/stash
I expect you need to put the path into the field too

**Pull request merged**